.circleci/config.yml

```diff
@@ -6,6 +6,7 @@ jobs:
     machine: true
     environment:
       - CONDA_PY: "27"
+      - PYTHONHTTPSVERIFY: 0
```

Comment:
Editing CI files directly is strongly discouraged. Please use `conda-forge.yml`'s `matrix` feature if you want to do this and re-render. Here's an example ( https://github.com/conda-forge/tk-feedstock/pull/20 ). Though honestly I don't understand why this is needed. 😕